tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -133,14 +133,13 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
   }
 
   ignore("known issue - multiline difference between CPU and GPU") {
-    // see https://github.com/rapidsai/cudf/issues/9620
+    // see https://github.com/NVIDIA/spark-rapids/issues/4170
     val pattern = "2$"
-    // this matches "2" but not "2\n" on the GPU
+    // this matches "2" and "2\n" but not "2\r" on the GPU
     assertCpuGpuMatchesRegexpFind(Seq(pattern), Seq("2", "2\n", "2\r", "2\r\n"))
```

Comment:
Do we want to split out the cases that work from the cases that do not so we don't over-ignore here?  For example, are we regularly testing something that ends in `\r\n` anywhere?